pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -20,11 +21,12 @@
 
 
 class PastisMatrixEfields(PastisMatrix):
-    instrument = None
 
     def __init__(self, design=None, initial_path='', saveefields=True, saveopds=True):
         super().__init__(design=design, initial_path=initial_path)
 
+        self.instrument = CONFIG_PASTIS.get('telescope', 'name')
+
```

Comment:
This is a change we cannot do because it will break things, so please revert. The `instrument` attribute gets set in the child class, exactly like you have it in `MatrixEfieldRST`, so there is no need to change anything.

Reply:
@davidbourgeois this still needs to be addressed.